packages/kbn-monaco/src/painless/lexer_rules/painless.ts

```diff
@@ -189,6 +180,3 @@ export const lexerRules = {
     ],
   },
 } as Language;
-
-monaco.languages.register({ id: ID });
```

Comment:
I noticed the xjson and painless languages were getting registered twice - once here (and in `lexer_rules/xjson`, respectively), and again in the `registerLexerRules` function. I assume this was a mistake, but wanted to call out here if not.